server/test/instant/reactive/session_test.clj

```diff
@@ -92,9 +94,8 @@
           (session/on-close store-conn socket)
           (session/on-close store-conn socket-2)
           (finally
-            (when (realized? eph-hz)
```

Comment:
What might be happening is that the delay is triggered, but when we get here it hasn't finished initializing, so `realized?` is false.